src/test/regress/expected/grant_role_from_non_maindb.out

```diff
@@ -0,0 +1,160 @@
+CREATE SCHEMA grant_role2pc;
+SET search_path TO grant_role2pc;
+set citus.enable_create_database_propagation to on;
+CREATE DATABASE grant_role2pc_db;
+\c grant_role2pc_db
+SHOW citus.main_db;
+ citus.main_db
+---------------------------------------------------------------------
+ regression
+(1 row)
+
+SET citus.superuser TO 'postgres';
+CREATE USER grant_role2pc_user1;
+CREATE USER grant_role2pc_user2;
+CREATE USER grant_role2pc_user3;
```

Comment:
Can we rename those three in order to test with role names that require proper quoting:

"grant_role2pc\'_user1"
"grant_role2pc\'_user2"
"grant_role2pc\'_user3"

Reply:
Fixed as you suggested